tests/unit/printver.tcl

```diff
@@ -1,6 +0,0 @@
-start_server {} {
```

Comment:
I think we can just delete this file? I don't think there is any real value in it anymore.

Reply:
But in the description they mentioned "Print version info before running the test
it was created as a development test tool just for the two info-fields redis_version and redis_git_sha1 (which is now server_git_sha1)."

Reply:
I suggest removing the printver.tcl file directly, it makes no sense now. There is no any function test in this file.

Reply:
Okay will do that 👍